spread.yaml

```
@@ -0,0 +1,28 @@
+project: classic-snap
+
+path: /home/spread
+
+backends:
+  qemu:
+    systems:
+      - ubuntu-16.04-64:
```

How about adding 14.04, just in case?
```suggestion
      - ubuntu-14.04-64:
           username: ubuntu
           password: ubuntu
      - ubuntu-16.04-64:
```

---

I will try with 14.04, might be more work because of the new kernel and reboot etc. but worth a shoot for sure :)

---

Oh,  for 14.04 I just use a qemu image with the newer kernel. No need to do it in the suite IMO.